planet/src/main/webapp/places.js

```diff
@@ -339,35 +349,32 @@ function listResults() {
 
 /** Toggle save icon on click */
 function savePlace(x) {
+    let placeID = $(x).attr('name');
+    let userID = 'userID';
     if(x.innerHTML === "favorite_border") {
         // Set as saved
         x.innerHTML = "favorite";
         // Save place to database
-        let name = document.getElementById('place-name').innerHTML; 
-        let address = document.getElementById('place-address').innerHTML; 
-        let open = document.getElementById('openingTime').innerHTML;
-        let close = document.getElementById('closingTime').innerHTML;
-        updateDatabase('test', name, address, open, close); 
+        updateDatabase(placeID); 
     }
     else {
         // Set as unsaved
         x.innerHTML = "favorite_border";
         // Delete place from database
-        deletePlace('test');
+        deletePlace(placeID, userID);
     }
 }
 
 /** Update database and add place with information */
-function updateDatabase(placeID, name, address, open, close) {
-    database.ref('places/' + placeID).set({
-        name: name,
-        address: address,
-        openingTime: open,
-        closingTime: close
+function updateDatabase(placeID) {
+    // Add placeID with userID (temporary variable)
+    database.ref('places/' + placeID + '/users').set({
+        user_ID: 'userID'
     });
 }
 
-/** Delete place from database when unsaved by user */
-function deletePlace(placeID) {
-    database.ref('places/' + placeID).remove(); 
+/** Delete user from places when unsaved by user */
+function deletePlace(placeID, userID) {
+    database.ref('places/' + placeID + '/users' + userID).remove();
+    console.log('removed');
```

Comment:
It looks like the path format for the update and the delete are different and that the delete won't actually delete the entry added at line 371. 

I think additional testing might be necessary here.